planner/core/physical_plans.go

```diff
@@ -1058,7 +1058,7 @@ func (p *basePhysicalAgg) getAggFuncCostFactor() (factor float64) {
 		}
 	}
 	if factor == 0 {
-		factor = 1.0
+		factor = 0.1
```

Comment:
Maybe you could add some comments and directly use `aggFuncFactor[firstrow]`.

Reply:
ok, using `aggFuncFactor[firstrow]` is a good idea!